ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -109,11 +109,20 @@ void RSkComponentTextInput::OnPaint(SkCanvas *canvas) {
                           std::make_shared<skia::textlayout::ParagraphBuilderImpl>(
                           paraStyle, data.layoutManager->collection_));
 
+  std::string securedText{};
+  int displayStrSize = displayString_.size();
+  if (textInputProps.secureTextEntry) {
+    std::string secureTextEntryText = "*";
```

Comment:
see if you can make use of already available functions for string replace like std::replace , boost::replace_all_copy